packages/ibm-products/src/components/InterstitialScreenViewModule/InterstitialScreenViewModule.tsx

```diff
@@ -33,10 +33,27 @@ const componentName = 'InterstitialScreenViewModule';
 // Default values should be provided when the component needs to make a choice
 // or assumption when a prop is not supplied.
 
+interface InterstitialScreenViewModuleProps extends PropsWithChildren {
```

Comment:
This component doesn't use `children` so don't think you have to extend `PropsWithChildren` generic type.